File: app/src/main/java/com/frank/ffmpeg/listener/OnHandleListener.java
package com.frank.ffmpeg.listener;

/**
 * listener of FFmpeg processing
 * Created by frank on 2019/11/11.
 */
public interface OnHandleListener {
    void onBegin();
    default void onProgress(int progress, int duration){}
    void onEnd(int resultCode, String resultMsg);
}

File: app/src/main/java/com/frank/ffmpeg/listener/OnHandleListener.kt
package com.frank.ffmpeg.listener

/**
 * listener of FFmpeg processing
 * Created by frank on 2019/11/11.
 */
interface OnHandleListener {
    fun onBegin()
    fun onMsg(msg: String)
    fun onProgress(progress: Int, duration: Int)
    fun onEnd(resultCode: Int, resultMsg: String)
}
